tests/process_test.c

```diff
@@ -28,3 +28,26 @@ static int s_get_pid_sanity_check_test_fn(struct aws_allocator *allocator, void
 }
 
 AWS_TEST_CASE(get_pid_sanity_check_test, s_get_pid_sanity_check_test_fn)
+
+#ifdef _WIN32
+AWS_STATIC_STRING_FROM_LITERAL(s_test_command, "echo {\"Version\": 1, \"AccessKeyId\": \"AccessKey123\"}");
+#else
+AWS_STATIC_STRING_FROM_LITERAL(s_test_command, "echo '{\"Version\": 1, \"AccessKeyId\": \"AccessKey123\"}'");
+#endif
+AWS_STATIC_STRING_FROM_LITERAL(s_expected_output, "{\"Version\": 1, \"AccessKeyId\": \"AccessKey123\"}");
+
+static int s_run_command_test_fn(struct aws_allocator *allocator, void *ctx) {
+    (void)ctx;
+    struct aws_run_command_options options = {.allocator = allocator, .command = aws_string_c_str(s_test_command)};
+
+    struct aws_run_command_result *result = aws_run_command(options);
+    ASSERT_NOT_NULL(result);
+
+    ASSERT_BIN_ARRAYS_EQUALS(
+        result->std_out->bytes, result->std_out->len, s_expected_output->bytes, s_expected_output->len);
+
+    aws_run_command_result_destroy(result);
+    return AWS_OP_SUCCESS;
+}
+
+AWS_TEST_CASE(run_command_test, s_run_command_test_fn)
```

Comment:
Can we add a case that runs a command that generates an error?

Reply:
Will do